include/glow/Runtime/HostManager/HostManager.h

```diff
@@ -56,13 +56,12 @@ class HostManager final {
   /// refuse additional request and return Failed.
   const unsigned int activeRequestLimit_ = 20;
 
-  /// A map from a networkName to the DAG that represents the root of a network.
-  std::unordered_map<std::string, std::unique_ptr<DAGNode>> roots_;
+  /// A map from a networkName to a network, which is represented by a pair
+  /// <root, the rest nodes of the network>.
+  std::unordered_map<std::string, std::pair<rootDAGNodeTy, nodesDAGNodeTy>>
```

Comment:
If we have a DAG struct this would just be 
`std::unordered_map<std::string, DAG> networks_`